nautobot/core/templates/inc/media_failure.html

```diff
@@ -0,0 +1,111 @@
+{% load static %}
+
+<script>
+    (function initMediaFailure() {
+        // Create React-style `ref` reference to the media failure alert element.
+        const failureAlertRef = { current: null };
+
+        // Store all media failures inside `Map` containing `Set` values to ensure uniqueness of all reported failures.
+        const failures = new Map();
+
+        function handleFailure(element) {
+            // Get media `filename` by stripping down `href` or `src` element attribute of its `base` (static) path and query params.
+            const base = `${window.location.origin}{% static '' %}`;
+            const filename = (element.href || element.src || '').replace(base, '').replace(/\?.*$/, '');
```

Comment:
I wonder if this stripping is desirable/necessary. Particularly in the production deployment use case, wouldn't we *want* the full URL?